test/main.js

```diff
@@ -48,6 +48,9 @@ function init(callback) {
 		this.seed = this.startingSeed = [0x09d56, 0x08642, 0x13656, 0x03653];
 	};
 
+	// Disable writing to modlog
+	require('./../command-parser.js').CommandContext.prototype.logModCommand = noop;
```

Comment:
Hmm I have seen one such "bad file descriptor" error before in certain machines, but I thought it was caused by a Mock-FS bug. I wonder what's going on.